packages/cli/test/unit/commands/pull.test.ts

```diff
@@ -65,6 +67,68 @@ describe('pull', () => {
     await expect(exitCodePromise).resolves.toEqual(1);
   });
 
+  it('should link an unlinked project with a git config without connecing a git provider', async () => {
```

Comment:
This test got messy. Part of it is that we want to test that something doesn't happen, but the other part is that our API mocking makes it hard to do anything uncommon.

We may not really need this test. If we want to keep it, I could clean it up a bit more.